package.json

```diff
@@ -186,6 +186,10 @@
     "globals": {
       "AVAILABLE_STAGING_LOCALES": null
     },
+    "snapshotFormat": {
+      "escapeString": true,
+      "printBasicPrototype": true
+    },
```

Comment:
The new jest defaults to `false` for these values. This makes it necessary to regenerate all snapshots.

I like `printBasicPrototype: false` because this makes the snapshots more easily copy-pastable, but not really `escapeString: false` because the resulting strings are not valid.

But for now I set it both to true so that our snapshots stay the same, because I don't want to make the life of our contributor more complicated.
What do you think?

Reply:
It sounds good to me. When we have less PRs in progress, we can revisit this. Would you like to open an issue about it?

Reply:
Sure, I filed #4215